### PR TITLE
fix(circle): correct circle subscription input

### PR DESCRIPTION
**Pull request by devformatters**

### CODE REVIEW

File: src/types/circle.ts [hunk 1]
```diff
@@ -172,6 +172,11 @@ export default /* GraphQL */ `
     targetId: ID!
   }
 
+  input CircleSubscriptionInput {
+    "Unique ID."
+    id: ID!
```
Comment (robertu7) [inline src/types/circle.ts:177]: missing `password: String` for further subscriptions?

Comment (robertu7) [reply src/types/circle.ts:177]: related discussion: https://mattersnews.slack.com/archives/C01CYECRYTH/p1609814350006600

**Review by robertu7 — CHANGES_REQUESTED** [review]

File: src/types/circle.ts (hunk 1 above)
Comment (devformatters) [reply src/types/circle.ts:185]: Added. Thanks.

File: src/types/circle.ts [hunk 2]
```diff
@@ -172,6 +172,19 @@ export default /* GraphQL */ `
     targetId: ID!
   }
 
+  input SubscribeCircleInput {
+    "Unique ID."
+    id: ID!
+
+    "Wallet password."
+    password: String!
```
Comment (robertu7) [inline src/types/circle.ts:180]: Is password required for the first subscription?

**Review by robertu7 — APPROVED** [review]

File: src/types/circle.ts (hunk 2 above)
Comment (devformatters) [reply src/types/circle.ts:180]: I'll merge it first, and then we can discuss it in the following works.

**Pull request merged**